index.html

```diff
@@ -1,54 +1,187 @@
+      <div class="program-header">
+        <h2>Who Are iDreality Participants?</h2>
+        <hr />
+      </div>
+      <div class="program-list">
+        <div class="program-list-items">
+          <i class="fa-solid fa-users icon-bar"></i>
+          <h2>Ideator</h2>
+          <p>
+            This user joins the forum and shares daily problems faced at job,
+            home, or anywhere. The user helps the Dorks to innovate faster,
+            smarter, and easier
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-user-doctor icon-bar"></i>
+          <h2>Dork</h2>
+          <p>
+            This user is a techie. Dork connotes an expert or enthusiast
+            obsessed with a hobby in programming. The user eats and breathes
+            code
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-people-carry-box icon-bar"></i>
+          <h2>Aid</h2>
+          <p>
+            These are the users of development assistance to iDreality. They
+            help in the development of iDreality site. They also help in handy
+            with donations
+          </p>
+        </div>
+        <div class="program-list-items">
+          <i class="fa-solid fa-user-tie icon-bar"></i>
+          <h2>Financier</h2>
+          <p>
+            This user is a sponsor. The user introduces and supports a
+            proposal for implementation. The user supports the Ideator and the
+            Dork financially to actualize their project.
+          </p>
+        </div>
+        <div class="program-btn">
+          <button type="button">Join A Group</button>
+        </div>
+      </div>
+
+      <div class="program-btn-desktop">
+        <p><a href="#">Join a Group</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="experts">
+    <div class="expert-header">
+      <h2>Featured Contributors</h2>
+      <hr />
+    </div>
+
+    <div class="expert-list"></div>
+    <div class="experts-btn">
+      <button id="show-more-btn" type="button">
+        MORE
+        <i class="fa-solid fa-chevron-down icon-red"></i>
+      </button>
+    </div>
+  </section>
+
+  <section class="partners">
+    <div class="partners-header">
+      <h2>Partners</h2>
+      <hr />
+    </div>
+    <div class="partners-list">
+      <div class="partners-list-row-one">
+        <img src="./images/saf.png" alt="Safaricom" />
+        <img src="./images/airtel.png" alt="Airtel Kenya" />
+        <img src="./images/git.png" alt="GITHUB" />
+      </div>
+      <div class="partners-list-row-two">
+        <img src="./images/paypal.png" alt="PayPal" />
+        <img src="./images/microverse.png" alt="Microverse" />
+      </div>
+    </div>
+  </section>
+  <footer class="section-footer">
+    <div class="footer-left">
+      <a href="index.html"><img src="./images/logo.png" alt="Logo" /></a>
+    </div>
+
+    <div class="footer-right">
```

Comment:
- [ ] Please make sure the footer is in the center and follow the Behance design.

## see the image for reference
![image](https://user-images.githubusercontent.com/83803414/157932501-f4ecfbec-c40c-4473-a228-e1e7a8f865ee.png)